presto-verifier/src/main/java/com/facebook/presto/verifier/checksum/ChecksumResult.java

```diff
@@ -75,6 +75,11 @@ public Map<String, Object> getChecksums()
         return checksums;
     }
 
+    public boolean hasChecksum(String columnName)
```

Comment:
Looks like not used in this PR?

Reply:
Ah, yes, I used 1st and then had to go different path because of the NULLs corner case. :(
Took me so much time with the slow testing (build pkg).